vehicle/OVMS.V3/components/ovms_webserver/src/ovms_webserver.h

```diff
@@ -38,10 +38,17 @@
 #include <memory>
 #include <utility>
 #include <map>
+#include <string.h>
+#include <stdio.h>
+#include <string>
+#include <sstream>
+#include <fstream>
+#include <dirent.h>
```

Comment:
Includes not needed for the class API definition shall be placed in the source module instead, so modules only needing the class API don't need to pull in unnecessary additional headers.